src/components/services/DeepLinkingManager.js

```diff
@@ -28,23 +29,24 @@ export function DeepLinkingManager(props: Props) {
     requestAnimationFrame(() => dispatch(retryPendingDeepLink()))
   }, [accountReferralLoaded, dispatch, pendingDeepLink, wallets])
 
+  const handleUrl = url => {
+    try {
+      dispatch(launchDeepLink(parseDeepLink(url)))
+    } catch (error) {
+      showError(error)
+    }
+  }
```

Comment:
Why was this moved outside of the 'useAsyncEffect'?
It shouldn't matter too much but I think it's better if you put it back